examples/rails_example/app/views/home/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<p>
+    See evaluations of the <code>demo_feature</code> feature flag based on different splitting rules below:<br/><br/>
+    <strong>Globally Enabled?</strong>:&nbsp;<%= @flag_globally_enabled %><br/>
+    <strong>Enabled For Current User?</strong>:&nbsp;<%= @flag_enabled_for_user_key %><br/>
+    <strong>Enabled for Custom Splitting Rule?</strong>:&nbsp;<%= @flag_enabled_for_custom_splitting_rule %><br/>
+<p>
```

Comment:
Kinda relevant to the review since this is the view that renders the results of the flag evaluations.